tests/tui.rs

```diff
@@ -27,3 +28,61 @@ async fn run_app_with_no_args() {
     let state = initialize(config_path.path().to_path_buf());
     let _app = App::new(state);
 }
+
+#[tokio::test]
+async fn quit_app_from_all_tabs() {
+    let config_path = tempdir().unwrap();
+    let state = initialize(config_path.path().to_path_buf());
+
+    // SQL Tab
+    let mut app = App::new(state);
+    let key = event::KeyEvent::new(event::KeyCode::Char('q'), event::KeyModifiers::NONE);
+    let app_event = AppEvent::Key(key);
+    app.handle_app_event(app_event).unwrap();
+    // Ideally, we figure out a way to check that the app actually quits
```

Comment:
It would be nice to figure out how to properly test that the app quit and not just that the state says it should quit.  But I think this is an improvement so moving forward with current approach.